.travis.yml

```diff
@@ -15,9 +15,6 @@ install:
 
 script:
   - tox
-  # Needed for coveralls (.coverage files in tox envs not available):
-  - inv develop
-  - inv test
```

Comment:
By the way, I think the original idea here was that files generated in `tox` runs would be stored under `.tox/`, but apparently that's not the case... kinda interesting.

Reply:
Yeah, I thought so too once. Looks like tox sets cwd.